src/vs/platform/windows/electron-main/windowsService.ts

```diff
@@ -343,6 +343,9 @@ export class WindowsService extends Disposable implements IWindowsService, IURLH
 		if (path.scheme === Schemas.file) {
 			shell.showItemInFolder(path.fsPath);
 		}
+		if (path.scheme === Schemas.userData) {
```

Comment:
@sandy081 suggest to rename `showItemInFolder(path...` to `showItemInFolder(resource...` and do a proper `if`/`else` switch here.